README.md

```diff
@@ -1,4 +1,10 @@
 [![Build Status](https://travis-ci.org/openmrs/openmrs-module-metadatamapping.svg?branch=master)](https://travis-ci.org/openmrs/openmrs-module-metadatamapping)
 
 openmrs-module-metadatamapping
-==============================
\ No newline at end of file
+==============================
+
+Provides an easy & explicit way for metadata within a system (encounter types, location, etc.) to be mapped to external vocabularies.
+
+The aim is to solve many of the problems around metadata management by making it easier for developers to refer to metadata, simplify the synchronisation of metadata cross systems, etc. 
+
+For more information, check https://wiki.openmrs.org/pages/viewpage.action?pageId=6619649.
```

Comment:
Ohh thanks for the great work,as i was reviewing through i also found this link that can also be referred too https://wiki.openmrs.org/display/docs/Metadata+Mapping+Module

Reply:
Thanks for the heads up! I've added that link in now as well.